matrix/splunk-otel-dotnet-metadata.yaml

```diff
@@ -0,0 +1,564 @@
+  - env: OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED
+    description: Whether the logs console exporter is activated. The default value is `false`.The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+  - env: OTEL_DOTNET_AUTO_LOGS_INCLUDE_FORMATTED_MESSAGE
+    description: Whether the log state have to be formatted. The default value is `false`.
+    default: 'false'
+    type: boolean
+    category: diagnostic logging
+instrumentations:
+  - keys:
+      - ASPNET
+    instrumented_components:
+      - name: ASP.NET Framework (.NET Framework)
+        supported_versions: See general requirements
+    description: MVC / WebApi (Only integrated pipeline mode supported). Metrics requires trace instrumentation.
+    stability: beta
+    support: community
+    dependencies:
+      - name: ASP.NET Instrumentation for OpenTelemetry
+        source_href: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet
+        package_href: https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNet
+        version: 1.0.0-rc9.9
+        stability: beta
+      - name: ASP.NET Telemetry HttpModule for OpenTelemetry
+        source_href: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule
+        package_href: https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule
+        version: 1.0.0-rc9.9
+        stability: beta
+    signals:
+      - traces: 
+      - metrics:
+          - id: http.server.duration_{bucket|count|sum}
+            instrument: Cumulative counters (histogram)
+            description: Duration of the inbound HTTP request, in the form of count, sum, and histogram buckets. This metric originates multiple metric time series, which might result in increased data ingestion costs.
+  - keys:
+      - ASPNETCORE
+    instrumented_components:
+      - name: ASP.NET Core
+        supported_versions: See general requirements
+    description: Metrics automatically activates `Microsoft.AspNetCore.Hosting.HttpRequestIn` spans.
+    stability: beta
+    support: community
+    dependencies:
+      - name: ASP.NET Core Instrumentation for OpenTelemetry .NET
+        source_href: https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore
+        package_href: https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNetCore
+        version: 1.5.1-beta.1
+        stability: beta
+    signals:
+      - traces: 
+      - metrics:
+          - id: http.server.duration_{bucket|count|sum}
+            instrument: Cumulative counters (histogram)
+            description: Duration of the inbound HTTP request, in the form of count, sum, and histogram buckets. This metric originates multiple metric time series, which might result in increased data ingestion costs.
+  - keys:
```

Comment:
Instrumentations sometimes have instrumentation-specific config, at least in the java world. It should be possible to define extra settings on the instrumentation level.

Reply:
7f71cbad1cb8c94667516bc96b02a99a8b2970e8